app/controllers/SecondaryWarehouseDetailsController.scala

```diff
@@ -37,10 +36,10 @@ import scala.concurrent.{ExecutionContext, Future}
 class SecondaryWarehouseDetailsController @Inject()(
                                          override val messagesApi: MessagesApi,
                                          sessionRepository: SessionRepository,
-                                         navigator: Navigator,
```

Comment:
I can see that navigator pattern was not used on the onSubmit def. I know it wasn't part of your change but do you know why that was?

Reply:
I think as it has no logic and the only page it can go to is the check your answers